main.go

```diff
@@ -276,19 +403,48 @@ func createMilestoneData(advance int, timeInterval string) map[string]milestone
 	return milestones
 }
 
-func createMilestones(baseURL string, token string, projectID string, milestones map[string]milestone) error {
+func createMilestones(baseURL string, token string, projectID string, milestones map[string]milestone, api string) error {
 	client := &http.Client{}
-	strURL := []string{baseURL, "/projects/", projectID, "/milestones"}
+	var strURL []string
+	switch {
+	case api == "gitlab":
+		strURL = []string{baseURL, "/projects/", projectID, "/milestones"}
+	case api == "github":
+		strURL = []string{baseURL, "/milestones"}
+	}
 	URL := strings.Join(strURL, "")
 	params := url.Values{}
 	for _, v := range milestones {
-		params.Set("title", v.Title)
-		params.Set("dueDate", v.DueDate)
-		req, err := http.NewRequest("POST", URL, strings.NewReader((params.Encode())))
-		if err != nil {
-			return err
+		var req *http.Request
+		switch {
+		case api == "gitlab":
+			params.Set("dueDate", v.DueDate)
+			params.Set("title", v.Title)
+			req, err := http.NewRequest("POST", URL, strings.NewReader((params.Encode())))
+			if err != nil {
+				return err
+			}
+			req.Header.Add("PRIVATE-TOKEN", token)
+		case api == "github":
+			params.Set("due_on", v.DueDate)
```

Comment:
Is this still needed if it's passed in the JSON anyway ?

Reply:
You're right - I'll remove that line